stackdriver-core/lib/stackdriver/core/async_actor.rb

```diff
@@ -199,6 +199,30 @@ def async_stop!
       end
 
       ##
+      # Block current thread until the async job is fully stopped.
```

Comment:
@dazuma is arguing we should make this method public public, in the same way that `AsyncWriter#wait_until_stopped` was public to begin with.